ocaml/fstar-lib/generated/FStar_MRef.ml

```diff
@@ -4,10 +4,10 @@ type ('a, 'b, 'r, 'p, 'h) p_pred = unit
 type ('uuuuu, 'uuuuu1, 'r, 'p) token = unit FStar_ST.witnessed
 let witness_token :
   'uuuuu 'uuuuu1 . ('uuuuu, 'uuuuu1) FStar_ST.mref -> unit -> unit =
-  fun m -> fun p -> ()
+  fun m -> fun p -> FStar_ST.gst_recall (); FStar_ST.gst_witness ()
 let recall_token :
   'uuuuu 'uuuuu1 . ('uuuuu, 'uuuuu1) FStar_ST.mref -> unit -> unit =
-  fun m -> fun p -> ()
+  fun m -> fun p -> FStar_ST.gst_recall ()
 type ('a, 'rel) spred = unit
-let recall : 'p . unit -> unit = fun uu___ -> ()
-let witness : 'p . unit -> unit = fun uu___ -> ()
\ No newline at end of file
+let recall : 'p . unit -> unit = fun uu___ -> FStar_ST.gst_recall ()
+let witness : 'p . unit -> unit = fun uu___ -> FStar_ST.gst_witness ()
```

Comment:
Both of these are really noops, but they are effectful type applications in F* and should not have been dropped